go/gosec/tls_ssl_blacklist/ssl.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
+  - id: do-not-use-ssl-v3
```

Comment:
I tend to prefer saying something like `ssl-v3-is-insecure`. Telling users to blindly not use something doesn't really give them any idea *why* they shouldn't use it.

Reply:
ok addressed!